error.js

```diff
@@ -1,3 +1,18 @@
+exports.customErrorHandler = (err, req, res, next)=>{
+    console.log(err, '<-- err')
+    if(err.message && err.status){
+      res.status(err.status).send({message : err.message})
+    } else {
+      next(err)
+    }
+  }
+exports.idErrorHandler = (err, req, res, next)=>{
+    if(err.code === '22P02'){
+      res.status(400).send({message : 'Bad request'})
+    } else {
+      next(err)
+    }
+  }
```

Comment:
Error handling has been set up nicely here! I would move your console log to the serverErrorHandler as a 'catch all', essentially if you end up in the 500 then you can investigate. It's common practice to use it there rather than leaving the log in at the top of chain - so you only see it console log if you haven't dealt with it 👍